slurmci-test.toml

```diff
@@ -47,7 +46,8 @@ gpu = [
   { file = "test/Ocean/Hydrostatic_Boussinesq/test_divergence_free.jl", slurmargs = ["--ntasks=3"], args = [] },
   { file = "test/Ocean/Hydrostatic_Boussinesq/test_divergence_free_IMEX.jl", slurmargs = ["--ntasks=3"], args = [] },
   { file = "examples/Atmos/dry_rayleigh_benard.jl", slurmargs = ["--ntasks=3"], args = [] },
-  { file = "experiments/GCM/heldsuarez.jl", slurmargs = ["--ntasks=3"], args = [] },
-  { file = "experiments/LES/risingbubble.jl", slurmargs = ["--ntasks=3"], args = [] },
-  { file = "experiments/LES/dycoms.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "experiments/Atmos_GCM/heldsuarez.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "experiments/Atmos_LES/risingbubble.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "experiments/Atmos_LES/dycoms.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "experiments/Ocean_BoxGCM/homogeneous_box.jl", slurmargs = ["--ntasks=3"], args = [] },
```

Comment:
How about `ocean_gyre.jl`?

Reply:
Unfortunately, we don't have a good test for `ocean_gyre.jl`. It really needs to start from a checkpoint file to perform a useful test